### ISSUE TITLE
Trouble finding rt73usb monitor mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Akash786mrn/wifite#23]
```
What steps will reproduce the problem?
1. Observe RT73USB chipset device in managed mode with 'iwconfig'
2. Launch ./wifite.py -e linksys -wepw 0
3. It finds my adapter and offers to place into monitor mode, but later it 
cannot find it.

What is the expected output? What do you see instead?
Monitor mode adapter found and functional for wifite.py

What version of the product are you using? On what operating system?
r43

Please provide any additional information below.
Start with:
root@bt4-final:~/captures# iwconfig
lo        no wireless extensions.

eth0      no wireless extensions.

rausb0    RT73 WLAN  ESSID:""
          Mode:Managed  Frequency=2.412 GHz  Bit Rate=54 Mb/s
          RTS thr:off   Fragment thr:off
          Encryption key:off
          Link Quality=0/100  Signal level:-121 dBm  Noise level:-143 dBm
          Rx invalid nwid:0  Rx invalid crypt:0  Rx invalid frag:0
          Tx excessive retries:0  Invalid misc:0   Missed beacon:0

Run wifite:
root@bt4-final:~/captures# ./wifite.py -e linksys -wepw 0
éèàù

  .;'                     `;,
 .;'  ,;'             `;,  `;,   WiFite r43
.;'  ,;'  ,;'     `;,  `;,  `;,
::   ::   :   ( )   :   ::   ::  mass WEP/WPA cracker
':.  ':.  ':. /_\ ,:'  ,:'  ,:'
 ':.  ':.    /___\    ,:'  ,:'   designed for backtrack4
  ':.       /_____\      ,:'
           /       \

[+] set wep attack wait time: unlimited

[+] searching for devices in monitor mode...
[!] no wireless interfaces are in monitor mode!

[+] select which device you want to put into monitor mode:
      1. rausb0         Ralink 2573 USB rt73
[+] select the wifi interface (between 1 and 1): 1
[+] putting "rausb0" into monitor mode...
[+] searching for devices in monitor mode...
[+] defaulting to interface ""

[+] waiting for "linksys" to appear, press Ctrl+C to skip...

[!] the program was unable to capture airodump packets!
[+] please make sure you have properly enabled your device in monitor mode
[+] the program is unable to continue and will now exit

The program terminated, but notice it placed the adapter into monitor mode:
root@bt4-final:~/captures# iwconfig
lo        no wireless extensions.

eth0      no wireless extensions.

rausb0    RT73 WLAN  ESSID:""
          Mode:Monitor  Frequency=2.457 GHz  Bit Rate=54 Mb/s
          RTS thr:off   Fragment thr:off
          Encryption key:off
          Link Quality=0/100  Signal level:-121 dBm  Noise level:-143 dBm
          Rx invalid nwid:0  Rx invalid crypt:0  Rx invalid frag:0
          Tx excessive retries:0  Invalid misc:0   Missed beacon:0

The bottom line is that wifite seems unable to find the adapter, for some 
strange reason. Are there literal strings it searches for but stumbles upon?

The "linksys" access point belongs to me, and yes it is beaconing at close 
range to my BT4 box.

Running BT4-final with python version 2.5.2 and Aircrack-ng 1.1 r1777.

I know that injection works great on my platform, and that WEP cracking is a 
breeze from the command line. Hoping that a fix to this will make wifite more 
appealing to a greater audience.

Thanks.

```

Original issue reported on code.google.com by `cpmitc...@gmail.com` on 2 Oct 2010 at 8:59

**Comment by GoogleCodeExporter** [comment]
```
Yikes!  I didn't know that "iwconfig" could output "Mode:Montior" on a separate 
line.

I think I fixed it in Revision 48. Please confirm when you have time to test.

FYI: the option "./wifite.py -upgrade" *should* upgrade wifite to the latest 
revision.
```

Original comment by `der...@gmail.com` on 2 Oct 2010 at 11:26
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Just had a chance to try out the fix... looks good. Thanks. I'll take it for a 
spin and let you know if any other issues pop up.

We can close this bug report.
```

Original comment by `cpmitc...@gmail.com` on 3 Oct 2010 at 3:10

**Comment by GoogleCodeExporter** [comment]
```
Thanks for submiting this. I would have never known!
I'm glad I was able to patch it with only 1 revision (a new record!).
```

Original comment by `der...@gmail.com` on 3 Oct 2010 at 6:13
- Changed state: **Verified**
